cime_config/testdefs/testlist_cam.xml

```diff
@@ -53,36 +36,27 @@
     </options>
   </test>
 
-<!-- NorESM-CAM6 combination not supported (yet?)
-
-  <test compset="NF1850" grid="ne30pg3_ne30pg3_mtn14" name="ERP_Ln9" testmods="cam/outfrq13m">
+  <test compset="NF1850oslo" grid="ne30pg3_ne30pg3_mtn14" name="ERP_Ln9" testmods="cam/outfrq9s">
     <machines>
-      <machine name="betzy" compiler="intel" category="prealpha"/>
+      <machine name="betzy" compiler="intel" category="aux_cam_noresm"/>
     </machines>
     <options>
       <option name="wallclock">00:20:00</option>
     </options>
   </test>
 
-  <test compset="NF2000climo" grid="ne30pg3_ne30pg3_mtn14" name="ERP_Ln9" testmods="cam/outfrq1d">
-    <machines>
-      <machine name="betzy" compiler="intel" category="prealpha"/>
-    </machines>
-    <options>
-      <option name="wallclock">00:20:00</option>
-    </options>
-  </test>
+  <!-- NorESM-CAM6 combination not supported (these are NorESM physics with CAM60)
 
-  <test compset="NF2000climo" grid="ne3pg3_ne3pg3_mg37" name="ERP_D_Ln9_P36" testmods="cam/outfrq9s">
+  <test compset="NF2000climoC6" grid="ne30pg3_ne30pg3_mtn14" name="ERP_Ln9" testmods="cam/outfrq1d">
     <machines>
-      <machine name="betzy" compiler="intel" category="aux_cam_noresm"/>
+      <machine name="betzy" compiler="intel" category="prealpha_noresm"/>
     </machines>
     <options>
       <option name="wallclock">00:20:00</option>
     </options>
   </test>
 
-  <test compset="NQPC2000climo" grid="ne3pg3_ne3pg3_mg37" name="ERP_D_Ln9_P36" testmods="cam/outfrq9s">
+  <test compset="NQPC2000climoC6" grid="ne3pg3_ne3pg3_mg37" name="ERP_D_Ln9_P36" testmods="cam/outfrq9s">
```

Comment:
Have you tried this testlist? I don't think that compsets `NQPC2000climoC6` or `NF2000climoC6` even exist in this branch. Please check that the `aux_cam_noresm` and `prealpha_noresm` testlists work.

Reply:
Again - I don't know what file you are looking at. This line is commented out in the testlist_cam.xml.

Reply:
I did not notice the comments by why are these here at all? What purpose do they serve?